docs/uglify-options.md

```diff
@@ -7,113 +7,94 @@ This task primarily delegates to [UglifyJS2][], so please consider the [UglifyJS
 
 
 ## mangle
-Type: `Boolean` `Object`  
```

Comment:
@alexlamsl: can you put back the trailing spaces in markdown files? They are used for line breaks.